.env.template

```diff
@@ -195,3 +195,9 @@ OPENAI_API_KEY=your-openai-api-key
 
 #ALLOWLISTED_PLUGINS - Sets the listed plugins that are allowed (Example: plugin1,plugin2,plugin3)
 ALLOWLISTED_PLUGINS=
+
+################################################################################
+### CHAT PLUGIN SETTINGS
+################################################################################
+# CHAT_MESSAGES_ENABLED - Enable chat messages (Default: False)
+# CHAT_MESSAGES_ENABLED=False
```

Comment:
Why would this be gated?

Reply:
I figured it's easier to check that way, so that the plugins don't get iterated everytime an interaction takes place...

Reply:
Also another thought:
You wouldn't want to keep deleting the plugins just to not use it for once.
Maybe a Plugin Manage would be nice